R/FilterStates.R

```diff
@@ -133,7 +133,16 @@ FilterStates <- R6::R6Class( # nolint
         simplify = FALSE,
         function(state_list) {
           items <- state_list()
-          filtered_items <- Filter(f = function(x) x$is_any_filtered(), x = items)
+          filtered_items <- Filter(
+            f = function(x) x$is_any_filtered() && !identical(attr(x, "sid"), sid),
+            x = items
+          )
+          # hierarchical
+          #filtered_items <- Filter(
+          #  f = function(x) x$is_any_filtered() && attr(x, "sid") < sid,
+          #  x = items
+          #)
```

Comment:
Uncomment to get hierarchical!

Reply:
@npaszty 🥳 (it'll be a long time until it's in main but thought you'd be happy)